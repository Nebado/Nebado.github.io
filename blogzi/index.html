<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8">
        <style>
         * {
             margin: 0px;
             padding: 0px;
         }
         body {
             width: 1110px;
             margin: auto;
             background: rgba(32,33,255,0.2);
         }
         h3, p {
             padding: 10px;
         }

         .post-item {
             border: 1px dotted #000;             
             background: rgba(33,123,33,.2);
         }
         .timestamp {
             text-align: right;
             font-style: italic;
         }
        </style>
    </head>
    <body>
        <h1 style="text-align: center; margin: 30px 0px; color: #181818;">Blog ZI</h1>

        <div class="post-item">
            <h3 class="post-title">#1 And again on September 3</h3>
            <p class="post-description">(((function thirdSeptember))).</p>            
            <p class="timestamp">03.09.2020</p>
        </div>
        
        <div class="post-item">
            <h3 class="post-title">#2 Some repair</h3>
            <p class="post-description">I repaired mp3 autoplayer. I have idea, write on own firmware on android phone.</p>
            <p class="timestamp">05.09.2020</p>
        </div>

        <div class="post-item">
            <h3 class="post-title">#3 Izya Day</h3>
            <p class="post-description">Today is an izya programmer day.</p>
            <p class="post-description">{{ PHP,JS }}. localStorage.setItem('level', 'Izya'); localStorage.getItem('level'). @extends {{ $default_layout }} -> @yield{{ content }}. {array}=>current{reset}(),end(); regex => "/^d+$/"; "/D/"; "[0-9]+";
            </p>
            <p class="timestamp">07.09.2020</p>
        </div>
    </body>
</html>
