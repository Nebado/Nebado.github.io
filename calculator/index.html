<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
	<title>Calculator JS</title>
	<link rel="stylesheet" href="css/fonts.css">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<div class="wrapper">
		<div class="bg_overlay">
			<div class="container">
				<div class="main clearfix">
					<div class="topline">
						<div class="logo">
							<h3>Calculator JS</h3>
						</div>
					</div>
					<div class="left">
						<h1>Calculator C-700</h1>
						<div class="calc">
							<div class="calc_title">C-700</div>
							<div class="calc_display">
								<input disabled type="text" id="display" value="0" class="calc__display-input" placeholder="000000000">
							</div>
							<div class="calc_buttons">
								<table class="calc__table">
									<tr class="calc__buttons-row">
										<td>
											<button type="button" id="ce" class="clear_btn btn btn-red">ce</button>
										</td>
										<td>
											<button type="button" id="c" class="clear_btn btn btn-red">c</button>
										</td>
										<td>
											<button type="button" class="btn btn-blue operation">%</button>
										</td>
										<td>
											<button value="Divide" type="button" class="btn btn-blue operation">/</button>
										</td>
									</tr>
									<tr class="calc__buttons-row">
										<td>
											<button type="button" class="btn number">7</button>
										</td>
										<td>
											<button type="button" class="btn number">8</button>
										</td>
										<td>
											<button type="button" class="btn number">9</button>
										</td>
										<td>
											<button value="Multiply" type="button" class="btn btn-blue operation">*</button>
										</td>
									</tr>
									<tr class="calc__buttons-row">
										<td>
											<button type="button" class="btn number">6</button>
										</td>
										<td>
											<button type="button" class="btn number">5</button>
										</td>
										<td>
											<button type="button" class="btn number">4</button>
										</td>
										<td>
											<button value="Substraction" type="button" class="btn btn-blue operation">-</button>
										</td>
									</tr>
									<tr class="calc__buttons-row">
										<td>
											<button type="button" class="btn number">1</button>
										</td>
										<td>
											<button type="button" class="btn number">2</button>
										</td>
										<td>
											<button type="button" class="btn number">3</button>
										</td>
										<td>
											<button value="Addition" type="button" class="btn btn-blue operation">+</button>
										</td>
									</tr>
									<tr class="calc__buttons-row">
										<td>
											<button type="button" class="btn btn-red">b</button>
										</td>
										<td>
											<button type="button" class="btn number">0</button>
										</td>
										<td>
											<button type="button" id="decimal" class="btn">.</button>
										</td>
										<td>
											<button type="button" id="result" class="operation btn btn-blue">=</button>
										</td>
									</tr>
								</table>
							</div>
						</div>
					</div>
					<div class="right">
						<h2>How does it works</h2>
						<button id="howWorkBtn" class="btn">How Works</button>
						<button id="removeWorkBtn" class="btn">Remove</button>
						<ul id="operationsList">
						</ul>	
					</div>
				</div>	
			</div>
		</div>
	</div>
	<script src="js/common.js" type="text/javascript"></script>
</body>
</html>
