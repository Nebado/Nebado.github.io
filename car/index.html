<!-- Author: Nebado -->
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
	<title>Car TimeLine</title>
	<link rel="stylesheet" href="libs/bootstrap/bootstrap-grid.min.css">
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/main.css">
	<link rel="stylesheet" href="css/media.css">
</head>
<body>
	<div class="container">
		<div class="row">
			<div class="col-md-3">
				<!-- Section of Years -->
				<div class="release_years">
					<h2 class="release_year">1987</h2>
					<h2 class="release_year">1988</h2>
					<h2 class="release_year active">1989</h2>
					<h2 class="release_year">1990</h2>
				</div>
			</div>
			<div class="col-md-9">
				<!-- Section of Cars -->
				<div class="cars_wrap">
					<div class="car_item">
						<div class="row">
							<div class="col-md-9">
								<div class="logo">
									<img src="img/logo.png" alt="Logo">
								</div>
								<h1>Porsche 964</h1>
								<div class="view_wrap">
									<span>1</span>
									<span>2</span>
									<span>3</span>
									<span>4</span>
								</div>
								<div class="car_item_img">
									<img src="img/car_porsche_964.png" alt="Porsche 964">
								</div>
							</div>
							<div class="col-md-3">
								<div class="years_wrap">
									<h2>1994-</h2>
									<h1>1989</h1>
								</div>
								<div class="type_cars">
									<div class="divider"></div>
									<p class="type_header">Class</p>
									<p class="type_car">Sportcars</p>
									<div class="divider"></div>
								</div>
								<div class="button_wrap">
									<button type="button" class="btn_more">Read <br>more</button>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>

	<script src="libs/jquery/jquery-3.3.1.min.js" type="text/javascript"></script>
	<script src="libs/bootstrap/bootstrap.min.js" type="text/javascript"></script>
	<script src="js/common.js" type="text/javascript"></script>
</body>
</html>
