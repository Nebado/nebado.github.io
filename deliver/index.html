<!-- Author: Nebado -->
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Deliver</title>
    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
    <link rel="stylesheet" href="css/font-face.css">
    <link rel="stylesheet" href="css/main.css">
  </head>
  <body>
    <!-- Header -->
    <header class="header">
      <div class="container">
        <div class="header__row">
	  <div class="top__menu">
	    <button class="lang"><span>English <i class="fa fa-angle-down"></i></span></button>
	    <div class="contact">
	      <button><i class="fa fa-users"></i> Contact Center</button>
	      <button>Country Profile <i class="fa fa-angle-down"></i></button>
	    </div>
	  </div>
        </div>
      </div>
      <div class="main__menu" id="main-menu">
        <div class="container"> 
          <div class="main__menu-flex">
            <input class="menu-btn" type="checkbox" id="menu-btn" />
            <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
            <div class="logo"><img src="img/logo.png" alt="logo"></div>
	    <ul class="menu">
	      <li><a href="#">Express</a></li>
	      <li><a href="#">Parcel & Ecommerce</a></li>
	      <li><a href="#">Logistics</a></li>
	      <li><a href="#">Mail</a></li>
	      <li><a href="#">Press</a></li>
	      <li><a href="#">About</a></li>
	    </ul>
	    <div class="search"><button><i class="fa fa-search"></i></button></div>
          </div>
        </div>
      </div>
    </header>
    <!-- Slider -->
    <div class="slider">
      <div class="container">
        <div class="slider__title">
	  <h1>We Deliver <br>With Speed</h1>
	  <p>When an unknown printer took a gallery of type and scrambled <br>
	    it to make a type specimen book. It has survived not only five centuries</p>
        </div>
        <div class="slider__features">
	  <p>Amazing  <span>.</span>  Quick  <span>.</span>  24/7 Supports</p>
        </div>
        <div class="slider__btn">
	  <button class="btn btn-1">Learn more</button>
	  <button class="btn btn-2">DHL Worldwide</button>
        </div>
      </div>
    </div>
    <!-- Services -->
    <div class="services">
      <div class="container">
	<div class="services__title">
	  <h1>Services</h1>
	  <h2>Awesome Services</h2>
	</div>
	<div class="services__items">
	  <div class="service__item">
	    <img src="img/service-item1.jpg" alt="">
	    <h3>Express Services</h3>
	    <p>When and unknown printer took a gallery of type and scrambled it to make a type specimen.</p>
	    <a href="#" class="more">Read more <i class="fa fa-caret-right"></i></a>
	  </div>
	  <div class="service__item">
	    <img src="img/service-item2.jpg" alt="">
	    <h3>Parcel & Ecommerce</h3>
	    <p>When and unknown printer took a gallery of type and scrambled it to make a type specimen.</p>
	    <a href="#" class="more">Read more <i class="fa fa-caret-right"></i></a>
	  </div>
	  <div class="service__item">
	    <img src="img/service-item3.jpg" alt="">
	    <h3>Frieght Transport</h3>
	    <p>When and unknown printer took a gallery of type and scrambled it to make a type specimen.</p>
	    <a href="#" class="more">Read more <i class="fa fa-caret-right"></i></a>
	  </div>
	  <div class="service__item service__location">
	    <h3>DHI Worldwide</h3>
	    <p>When an unknown prot printer type and scra</p>
	    <form action="#" class="location">
	      <div class="input__loc">
		<input type="text" placeholder="Location">
		<button type="submit" class="btn btn-loc"><i class="fa fa-angle-down"></i></button>
	      </div>
	      <button type="submit" class="explore">Explore</button>
	    </form>
	  </div>
	</div>
      </div>
    </div>
    <!-- About -->
    <div class="about">
      <div class="bg-overlay"></div>
      <div class="container">
	<div class="about__title">
	  <h1>One of the best international service provider</h1>
	  <button class="btn btn-about"><i class="fa fa-play"></i></button>
	</div>
	<div class="about__items">
	  <div class="about__item">
	    <img src="img/provider-item1.jpg" alt="">
	    <div class="about__item--title">
	      <h4>Aerospace Provider</h4>
	      <p><i class="fa fa-caret-right"></i>When an unknown printer took.</p>
	      <p><i class="fa fa-caret-right"></i>Make a type specimen book.</p>
	      <p><i class="fa fa-caret-right"></i>It has survived.</p>
	      <p><i class="fa fa-caret-right"></i>Airlines unknown specimen.</p>
	    </div>
	  </div>
	  <div class="about__item">
	    <img src="img/provider-item2.jpg" alt="">
	    <div class="about__item--title">
	      <h4>Poppins Proved</h4>
	      <p><i class="fa fa-caret-right"></i>When an unknown printer took.</p>
	      <p><i class="fa fa-caret-right"></i>Make a type specimen book.</p>
	      <p><i class="fa fa-caret-right"></i>It has survived.</p>
	      <p><i class="fa fa-caret-right"></i>Airlines unknown specimen.</p>
	    </div>
	  </div>
	  <div class="about__item">
	    <img src="img/provider-item3.jpg" alt="">
	    <div class="about__item--title">
	      <h4>Airline Specimen</h4>
	      <p><i class="fa fa-caret-right"></i>When an unknown printer took.</p>
	      <p><i class="fa fa-caret-right"></i>Make a type specimen book.</p>
	      <p><i class="fa fa-caret-right"></i>It has survived.</p>
	      <p><i class="fa fa-caret-right"></i>Airlines unknown specimen.</p>
	    </div>
	  </div>
	</div>
      </div>
    </div>
    <!-- Express -->
    <div class="express">
      <div class="container">
	<div class="express__block">
	  <div class="express__form">
	    <div class="filters" id="filters">
	      <span class="exp__tab">Express</span>
	      <span class="log__tab">Logistics</span>
	    </div>
	    <div class="express__accordion">
	      <div class="set">
		<a href="#express">Track your shipment<span><i class="fa fa-angle-down"></i></span></a>
		<div class="content">
		  <h5>International Services to Another Country</h5>
		  <p><i class="fa fa-caret-right"></i>Emergency</p>
		  <p><i class="fa fa-caret-right"></i>Time Critical</p>
		  <p><i class="fa fa-caret-right"></i>Less Urgent</p>
		</div>
	      </div>
	      <div class="set">
		<a href="#express">New login<span><i class="fa fa-angle-down"></i></span></a>
		<div class="content">
		  <h5>International Services to Another Country</h5>
		  <p><i class="fa fa-caret-right"></i>Emergency</p>
		  <p><i class="fa fa-caret-right"></i>Time Critical</p>
		  <p><i class="fa fa-caret-right"></i>Less Urgent</p>
		</div>
	      </div>
	      <div class="set">
		<a href="#express">Find DHL point<span><i class="fa fa-angle-down"></i></span></a>
		<div class="content">
		  <h5>International Services to Another Country</h5>
		  <p><i class="fa fa-caret-right"></i>Emergency</p>
		  <p><i class="fa fa-caret-right"></i>Time Critical</p>
		  <p><i class="fa fa-caret-right"></i>Less Urgent</p>
		</div>
	      </div>
	      <div class="set">
		<a href="#express">Find a service<span><i class="fa fa-angle-up"></i></span></a>
		<div class="content">
		  <h5>International Services to Another Country</h5>
		  <p><i class="fa fa-caret-right"></i>Emergency</p>
		  <p><i class="fa fa-caret-right"></i>Time Critical</p>
		  <p><i class="fa fa-caret-right"></i>Less Urgent</p>
		</div>
	      </div>
	    </div>
	  </div>
	  <div class="express__team">
	    <img src="img/express-img.png" alt="team">
	  </div>
	</div>
      </div>
    </div>
    <!-- Features -->
    <div class="features">
      <div class="container">
	<div class="features__block">
	  <div class="features__title">
	    <h2>Excellence. Simply Delivered.</h2>
	    <p>International express deliveries; global freight forwarding by air, sea, road and rail; warehousing solutions from packaging, to repairs, to storage; mail deliveries worldwide; and other customized logistic services - with everything DHL does, we <br> help connect people and improve their lives.</p>
	  </div>
	  <button type="button" class="btn btn-feat">Learn more</button>
	</div>
      </div>
    </div>
    <div class="clients">
      <div class="container">
	<div class="clients__block">
	  <div class="client__item"><img src="img/client1.jpg" alt=""></div>
	  <div class="client__item"><img src="img/client2.jpg" alt=""></div>
	  <div class="client__item"><img src="img/client3.jpg" alt=""></div>
	  <div class="client__item"><img src="img/client4.jpg" alt=""></div>
	</div>
      </div>
    </div>
    <footer id="footer">
      <div class="container">
	<div class="footer-info">
	  <div class="footer-menu contact-center">
	    <h5>Contact center</h5>
	    <ul>
	      <li><a href="#"><i class="fa fa-caret-right"></i>DHL Express</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>DHL Parcel & Ecommerce</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>DHL Global Forwarding</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>DHL Freight</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>DHL Supply Chain</a></li>
	    </ul>
	  </div>
	  <div class="footer-menu about-us">
	    <h5>About us</h5>
	    <ul>
	      <li><a href="#"><i class="fa fa-caret-right"></i>Partnerships</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>Company portrait</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>DHL Global Forwarding</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>Green Solutions</a></li>
	      <li><a href="#"><i class="fa fa-caret-right"></i>Innovation</a></li>
	    </ul>
	  </div>
	  <div class="footer-menu social">
	    <h5>Social Media</h5>
	    <ul>
	      <li><a href="#" class="facebook"><i class="fa fa-caret-right"></i>Facebook</a></li>
	      <li><a href="#" class="dribble"><i class="fa fa-caret-right"></i>Dribble</a></li>
	      <li><a href="#" class="behance"><i class="fa fa-caret-right"></i>Behance</a></li>
	      <li><a href="#" class="gmail"><i class="fa fa-caret-right"></i>Gmail</a></li>
	      <li><a href="#" class="twitter"><i class="fa fa-caret-right"></i>Twitter</a></li>
	    </ul>
	  </div>
	</div>
      </div>
    </footer>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    <script src="js/main.js"></script>
  </body>
</html>
