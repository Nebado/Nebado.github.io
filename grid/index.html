<!-- Author: Sorax -->
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="width" content="device-width, initial-scale=1.0">
	<title>Document</title>
	<link rel="stylesheet" type="text/css" href="css/main.css">
</head>
<body>

<div class="container">
	<div class="row">
		<div class="header span3">
			<h1>CSS Grid</h1>
		</div>
		<div class="search span3 offset6"><input type="text" name="" id=""></div>
	</div>
	<div class="row">
		<div class="slider span12"></div>
	</div>
	<div class="row">
		<div class="sidebar span2">
			<nav>
				<ul>
					<li><a href="">Item 1</a></li>
					<li><a href="">Item 2</a></li>
					<li><a href="">Item 3</a></li>
					<li><a href="">Item 4</a></li>
					<li><a href="">Item 5</a></li>
					<li><a href="">Item 6</a></li>
				</ul>
			</nav>
		</div>
		<div class="content span10">
			<h2>Lorem ipsum dolor sit amet.</h2>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nisi molestias voluptatum dolores minima vel velit in dolorum error mollitia, temporibus ad, tenetur tempora aliquid! Magni cumque officia voluptate eveniet quasi, labore officiis obcaecati eligendi, qui dicta itaque, debitis ipsum reprehenderit! Vitae doloremque adipisci atque labore blanditiis, voluptatibus accusantium corrupti ab esse praesentium dolorem ad, id asperiores fuga officiis, neque quis incidunt perferendis ipsa inventore. In fugiat, veritatis doloremque nisi! Blanditiis.</p>
			<p>Dicta inventore nihil debitis eius facere nulla a blanditiis magnam accusamus, placeat excepturi deleniti, consequuntur aliquid molestias aperiam? Rerum corporis sed quo unde eligendi accusamus tempora officia, doloremque velit beatae tenetur et distinctio eveniet similique cumque nobis quibusdam praesentium numquam, esse earum fugit vero impedit ipsa. Facilis maxime velit ratione at, libero a quaerat hic illo et veniam, officiis laudantium ducimus sequi debitis eos placeat cupiditate vitae iure fugiat pariatur.</p>
			<p>Aliquid distinctio reprehenderit corporis officia provident officiis ratione accusamus quae, suscipit, facilis quis excepturi recusandae quidem! Repellendus obcaecati fuga vitae qui culpa, temporibus deserunt autem sunt aliquid, natus fugit vero, dolorum earum! Blanditiis perspiciatis, dolor quam voluptas rem iure aliquam, asperiores eos corporis tempore in ullam placeat! Magni recusandae perspiciatis asperiores earum. Aliquid dicta id totam officiis porro, deserunt accusamus, distinctio doloribus dolore sed ducimus. Vel ratione incidunt beatae, harum.</p>
			<p>Perspiciatis quae saepe culpa doloremque, eius dignissimos dolores ab nisi. Commodi consequatur animi itaque, provident atque. Laboriosam cumque, quod odit eligendi tempore, fugiat tempora dolore quasi. Quisquam in voluptatibus eum quibusdam dolores necessitatibus recusandae fuga fugit, quae laboriosam, asperiores quia non at ullam! Qui neque culpa perferendis dolorum iure temporibus modi, consectetur molestiae, cum facilis rem quod natus, illo fugit autem, suscipit. Et tenetur unde natus nisi autem eum tempora!</p>
			<p>Earum sit fugiat quidem consequatur, voluptatibus quo tempore illum corporis illo tempora placeat nobis ipsa error excepturi quas quasi nisi consequuntur voluptatum. Consectetur at repellendus quas architecto sint dolor odio. Ipsa pariatur aliquid nemo, amet velit? Alias laborum eligendi itaque totam dignissimos molestiae beatae repudiandae quia cupiditate facilis recusandae qui dolorem atque quos deleniti, error excepturi adipisci! Tempora beatae corporis vel expedita, soluta. Non ad hic, blanditiis laboriosam debitis nulla!</p>
		</div>
	</div>
	<div class="row">
		<div class="footer span12"><small>Lorem ipsum dolor sit amet.</small></div>
	</div>
</div>

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" type="text/javascript"></script>
<script src="main.js" type="text/javascript"></script>
</body>
</html>
