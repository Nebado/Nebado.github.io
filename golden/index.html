<!-- Author: Nebado -->
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Golden</title>
	<link rel="stylesheet" type="text/css" href="css/reset.css">
	<link rel="stylesheet" type="text/css" href="css/style.css">
</head>
<body>
	<div id="wrapper">
		<header id="header">
			<div class="container clearfix">
				<div class="logo left"><a href="#"><img src="img/logo.png" alt="logo"></img></a></div>
				<ul class="nav right">
					<li><a href="#">Home</a></li>
					<li><a href="#">Services</a></li>
					<li><a href="#">Portfolio</a></li>
					<li><a href="#">About</a></li>
					<li><a href="#">Contact</a></li>
				</ul>
			</div>
		</header><!-- /header -->
		<div id="home">
			<div class="container">
				<h3>Welcome To Our Studio!</h3>
				<h1>Itâ€™s nice to Meet you</h1>
				<button type="button" class="btn">Tell me more</button>
			</div>
		</div>
		<div id="services">
			<div class="container">
				<h2 class="title">Services</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div id="services-wrap" class="clearfix">
					<div class="service-item commerce">
						<h5>E-Commerce</h5>
						<p>Proin iaculis purus consequat sem cure
						digni ssim. Donec porttitora entum suscipit</p>
					</div>
					<div class="service-item responsive">
						<h5>Responsive Web</h5>
						<p>Proin iaculis purus consequat sem cure
						digni ssim. Donec porttitora entum suscipit</p>
					</div>
					<div class="service-item security">
						<h5>Web Security</h5>
						<p>Proin iaculis purus consequat sem cure
						digni ssim. Donec porttitora entum suscipit</p>
					</div>
				</div>
			</div>
		</div>
		<div id="portfolio">
			<div class="container">
				<h2 class="title">Our Portfoliio</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div id="portfolio-wrap" class="clearfix">
					<div class="portfolio-item">
						<img src="img/p-item1.jpg" alt="item1">
						<div class="p-item-footer">
							<h5>Ebony & Ivory</h5>
							<span>Branding</span>
						</div>
					</div>
					<div class="portfolio-item">
						<img src="img/p-item2.jpg" alt="item1">
						<div class="p-item-footer">
							<h5>Smart Stationary</h5>
							<span>Print Design</span>
						</div>
					</div>
					<div class="portfolio-item">
						<img src="img/p-item3.jpg" alt="item1">
						<div class="p-item-footer">
							<h5>Clever Poster</h5>
							<span>Print Design</span>
						</div>
					</div>
					<div class="portfolio-item">
						<img src="img/p-item4.jpg" alt="item1">
						<div class="p-item-footer">
							<h5>Vinyl Record</h5>
							<span>Product Mock-Up</span>
						</div>
					</div>
					<div class="portfolio-item">
						<img src="img/p-item5.jpg" alt="item1">
						<div class="p-item-footer">
							<h5>Treehouse Template</h5>
							<span>Web Design</span>
						</div>
					</div>
					<div class="portfolio-item">
						<img src="img/p-item6.jpg" alt="item1">
						<div class="p-item-footer">
							<h5>Burned Logo</h5>
							<span>Branding</span>
						</div>
					</div>
				</div>
			</div>
		</div>
		<div id="about">
			<div class="container">
				<h2 class="title">About</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div id="about-wrap" class="clearfix">
					<div class="about-item left">
						<div class="about-i-text left text-right">
							<h5><span class="up">July 2010</span> <br>Our Humble Beginnings</h5>
							<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
							aenean rhoncus posuere odio in tincidunt. Proin
							iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-i-img left">
							<img src="img/about-item1.jpg" alt="">
						</div>
					</div>
					<div class="about-item right">
						<div class="about-i-text right text-left">
							<h5><span class="up">January 2011</span> <br>Facing Startup Battles</h5>
							<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
							aenean rhoncus posuere odio in tincidunt. Proin
							iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-i-img right">
							<img src="img/about-item2.jpg"  height="170px" alt="">
						</div>
					</div>
					<div class="about-item left">
						<div class="about-i-text left text-right">
							<h5><span class="up">December 2012</span> <br>Enter The Dark Days</h5>
							<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
							aenean rhoncus posuere odio in tincidunt. Proin
							iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-i-img left">
							<img src="img/about-item3.jpg"  height="170px" alt="">
						</div>
					</div>
					<div class="about-item right">
						<div class="about-i-text right text-left">
							<h5><span class="up">February 2014</span> <br>Our Triumph</h5>
							<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
							aenean rhoncus posuere odio in tincidunt. Proin
							iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-i-img right">
							<img src="img/about-item4.jpg" alt="">
						</div>
					</div>
					<div class="about-item right">
						<div class="about-i-img-a">
							<h4><a href="#">Our<br>Story<br>Continues<br>...</a></h4>
						</div>
					</div>
				</div>
			</div>
		</div>
		<div id="team">
			<div class="container">
				<h2 class="title">Our Amazing Team</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div id="team-wrap" class="clearfix">
					<div class="team-item left">
						<div class="team-img">
							<img src="img/t-item.png" alt="team">
						</div>
						<h5>Rico Massimo</h5>
						<span class="cat">Coder</span>
						<div class="social">
							<a href="#" class="ic-tw"></a>
							<a href="#" class="ic-fb"></a>
							<a href="#" class="ic-pn"></a>
						</div>
					</div>
					<div class="team-item left">
						<div class="team-img">
							<img src="img/t-item.png" alt="team">
						</div>
						<h5>Uku Mason</h5>
						<span class="cat">Graphic Designer</span>
						<div class="social">
							<a href="#" class="ic-tw"></a>
							<a href="#" class="ic-fb"></a>
							<a href="#" class="ic-gp"></a>
						</div>
					</div>
					<div class="team-item left">
						<div class="team-img">
							<img src="img/t-item.png" alt="team">
						</div>
						<h5>Kimberly Thompson</h5>
						<span class="cat">Marketer</span>
						<div class="social">
							<a href="#" class="ic-fb"></a>
							<a href="#" class="ic-pn"></a>
						</div>
					</div>
					<p class="tex">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
				</div>
			</div>
		</div>
		<div id="brands">
			<div class="container">
				<a href="#"><img src="img/envato.jpg" alt=""></a>
				<a href="#"><img src="img/wordpress.png" alt=""></a>
				<a href="#"><img src="img/tuts.jpg" alt=""></a>
				<a href="#"><img src="img/microlancer.jpg" alt=""></a>
			</div>
		</div>
		<div id="contacts">
			<div class="container">
				<h2 class="title white">Contact Us</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<form action="#" id="message">
					<div class="clearfix">
						<div class="left">
							<input type="text" placeholder="Your name *">
							<input type="text" placeholder="Your email *">
							<input type="text" placeholder="Subject">
						</div>
						<div class="right">
							<textarea cols="5" rows="5" placeholder="Your message *"></textarea>
						</div>
					</div>
					<input type="submit" value="SEND MESSAGE" class="btn">
				</form>
			</div>
		</div>
		<div id="copyright">
			<div class="container clearfix">
				<p class="left">&copy; Copyright 2019 FreebiesXpress.com</p>
				<div class="social left">
					<a href="#" class="ic-tw black">&nbsp;</a>
					<a href="#" class="ic-fb black">&nbsp;</a>
					<a href="#" class="ic-pn black">&nbsp;</a>
					<a href="#" class="ic-gp black">&nbsp;</a>
				</div>
				<div class="author right"><span>Nebado</span></div>
			</div>
		</div>
	</div>
</body>
</html>
